addons/l10n_it_edi/models/account_move.py

```diff
@@ -1031,117 +1028,65 @@ def _compose_info_message(self, tree, tags):
     # -------------------------------------------------------------------------
 
     def _l10n_it_edi_export_data_check(self):
-        errors = self._l10n_it_edi_base_export_data_check()
-        if not self._l10n_it_edi_is_simplified():
-            errors += self._l10n_it_edi_export_buyer_data_check()
+        errors = {
+            **self.mapped("company_id")._l10n_it_edi_export_check(),
+            **self._l10n_it_edi_base_export_check(),
+            **self._l10n_it_edi_export_taxes_check(),
+        }
+        for partner_kind, partners in {
+            'simplified': self.filtered(lambda move: move._l10n_it_edi_is_simplified()).mapped("commercial_partner_id"),
+            'other': self.filtered(lambda move: not move._l10n_it_edi_is_simplified()).mapped("commercial_partner_id"),
+            'representative': self.mapped("company_id.l10n_it_tax_representative_partner_id"),
+        }.items():
+            for key, error in partners._l10n_it_edi_export_check(partner_kind).items():
+                if key not in errors:
+                    errors[key] = error
+                else:
+                    errors[key]['action_params'] = list(set(errors[key]['action_params'] | error['action_params']))
         return errors
 
-    def _l10n_it_edi_format_export_data_errors(self):
```

Comment:
removing methods in stable?

Reply:
Yes, no one is developing on these methods.
Odoo Italian community is now migrating to 16.0, and they make PR to remove `l10n_it_edi` from `auto_install` because it interferes with the OCA ones.